pandas/io/tests/test_sql.py

```diff
@@ -2180,6 +2237,7 @@ def setUp(self):
         except:
             pass
         else:
+            self.addCleanup(self._try_close_db)
```

Comment:
For the tests to pass on my machine I need to close the database connection after each test.